async.c

```diff
@@ -681,10 +791,21 @@ int redisvAsyncCommand(redisAsyncContext *ac, redisCallbackFn *fn, void *privdat
     len = redisvFormatCommand(&cmd,format,ap);
 
     /* We don't want to pass -1 or -2 to future functions as a length. */
-    if (len < 0)
+    if (len == -1) {
```

Comment:
Might want to `#define`-away the meaning of -1, -2, etc.

Reply:
Might also want to do something like `redisvAsyncStrerror()` (static) rather than specify inline...

Reply:
I would like that too but currently these are defined by the redisvFormatCommand() 
If we would like to use a define we should change it there and delegate properly,
currently I don't want to go over the entire code base and see if this value was delegated somewhere and make sure we change it project-wide, probably should be a different commit/PR

Reply:
> Might also want to do something like `redisvAsyncStrerror()` (static) rather than specify inline...

Added something similar to that